packages/java/endpoint/src/main/java/dev/hilla/crud/PropertyStringFilterSpecification.java

```diff
@@ -75,6 +79,56 @@ public Predicate toPredicate(Root<T> root, CriteriaQuery<?> query,
             default:
                 break;
             }
+        } else if (isLocalDate(javaType)) {
```

Comment:
Not specifically related to this PR, but, are these `if-else`s only hurting my eyes or is it an epidemic?

Reply:
It seems that is it only hurting my eyes :)